src/engine/bank01.asm

```diff
@@ -7172,7 +7172,45 @@ Func_6cfa: ; 6cfa (1:6cfa)
 	ret
 ; 0x6d3f
 
-	INCROM $6d3f, $6d84
+Func_6d3f: ; 6d3f (1:6d3f)
+	or a
+	bit 7, [hl]
```

Comment:
This bit test (`bit 7`) is to see if the Pokemon is poisoned (and the `bit 6` bit test below is to see if the Pokemon is double poisoned).
The `7` and `6` correspond to `POISONED        EQU $80` and `DOUBLE_POISONED EQU $c0` respectively, from `src/constants/duel_constants.asm`

Reply:
You may want to create 2 new constants (`POISONED_F EQU 7` and `DOUBLE_POISONED_F EQU 6`) to go along with the existing `POISONED` and `DOUBLE_POISONED` constants.
(The `_F` suffix indicates that the constant represents a flag bit position, while the constant without the suffix is the literal value. ie, `$80` is the value when bit `7` is set.)